lib/data_update_scripts/20210119131811_replace_instances_of_banned_and_comment_banned.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class ReplaceInstancesOfBannedAndCommentBanned
+    def run
+      # Update names for the banned and comment_banned roles
+      # Details: https://github.com/forem/forem/pull/11581
+      Role.find_by(name: "banned").update(name: "suspended")
```

Comment:
Is every environment guaranteed to have these roles present? If not we should use a safety operator here and below.